pkg/arvo/app/azimuth.hoon

```diff
@@ -59,7 +59,10 @@
     ^-  (quip card _this)
     =.  net.state  %local
     :_  this
-    [%pass /eth-watcher %agent [our.bowl %eth-watcher] %watch /logs/[dap.bowl]]~
+    :~  [%pass /eth-watcher %agent [our.bowl %eth-watcher] %watch /logs/[dap.bowl]]
+        [%pass /old-tracker %agent [our.bowl %azimuth-tracker] %poke %kiln-nuke !>([%azimuth-tracker %base])]
```

Comment:
Shouldn't this be aimed at %hood?

Reply:
Most definitely! Not sure how this slipped in.